payjoin-io/src/ohttp_keys.rs

```diff
@@ -0,0 +1,89 @@
+use ohttp::KeyConfig;
+use tokio::task::spawn_blocking;
+use url::Url;
+
+use crate::error::{Error, InternalError};
+
+/// Returns the ohttp keys from a payjoin server.
+///
+/// `proxy_endpoint` - The proxy the user want to use when
+/// requesting the ohttp keys from a payjoin server.
+/// `pj_endpoint` - The payjoin server endpoint to fetch the ohttp
+/// keys from.
+pub async fn fetch(proxy_endpoint: Url, pj_endpoint: &Url) -> Result<KeyConfig, Error> {
```

Comment:
I imagine this naming came from payjoin-cli, however since this is going to be public API let's make sure to use conventional naming from [BIP 77](https://github.com/bitcoin/bips/pull/1483)

- `proxy_endpoint` is an `ohttp_relay` with CONNECT proxy enabled
- pj_endpoint is a `directory` server

By depending on `payjoin`, we can also make use of the `OhttpKeys` type exported there so we don't have to convert between the two. Return `Result<OhttpKeys, Error>` instead. No need to expose `ohttp` type when we have our own type that meshes with the `payjoin` library.